libraries/model-networking/src/model-networking/ModelCache.cpp

```diff
@@ -78,7 +80,7 @@ namespace std {
     struct hash<GeometryExtra> {
         size_t operator()(const GeometryExtra& geometryExtra) const {
             size_t result = 0;
-            hash_combine(result, geometryExtra.mapping, geometryExtra.textureBaseUrl, geometryExtra.combineParts);
+            hash_combine(result, geometryExtra.mapping.second, geometryExtra.textureBaseUrl, geometryExtra.combineParts);
```

Comment:
I wonder if we need to do this:
```suggestion
            hash_combine(result, geometryExtra.mapping.first, geometryExtra.mapping.second, geometryExtra.textureBaseUrl, geometryExtra.combineParts);
```
and you might need to add an std hash for QUrl